addons/web/static/src/legacy/debug_manager.js

```diff
@@ -21,6 +23,25 @@ class GetMetadataDialog extends Component {
         await this.getMetadata();
     }
 
+    async onClickCreateXmlid() {
+        const context = _.extend({}, this.context, {
+            'default_module': '__custom__',
+            'default_res_id': this.state.id,
+            'default_model': this.props.res_model,
+        });
```

Comment:
we don't use `_` in new code (+ no need to quote object keys in JS)
```suggestion
        const context = Object.assign({}, this.context, {
            default_module: '__custom__',
            default_res_id: this.state.id,
            default_model: this.props.res_model,
        });
```